protoclass/detection/extract_edge_volume.m

```diff
@@ -0,0 +1,61 @@
+function [edge_vol ] = extract_edge_volume( in_vol, Method, Threshold)
```

Comment:
Usually the variable does not have capital letter.

Modify `Method` for `method` and `Threshold` for `Threshold`.
Keep only the capital letter if this is a variable used by Mathworks inside one of their function and that we want to use the help.